src/main/java/errors/ErrorWritingDataToFile.java

```diff
@@ -0,0 +1,7 @@
+package errors;
+
+public class ErrorWritingDataToFile extends RuntimeException {
```

Comment:
```suggestion
public class WriteDataToFileException extends RuntimeException {
```

custom exception name should end with 'Exception'. fix in all similar places